unity/EditorPlugin/AfterUnity56/UnitTesting/TestEvent.cs

```diff
@@ -23,14 +23,16 @@ public TestEvent(EventType type, TestInternalEvent tEvent)
   public class TestInternalEvent
   {
     public string myID;
+    public readonly string myAssemblyName;
```

Comment:
Make the other field `readonly` too?

Reply:
Well spotted! I have to actually remove readonly here. Otherwise it will not get restored after AppDomain reload.